### PR TITLE
Fix #374

### LINKED ISSUE #374: [.5][ariesjia] Change Word and Calculation from "BALANCE" to "VALUE" on Top Bar
Ok, here is my argument:

1) Word: Under the Account tab you have "Balances". Balances are just which assets you hold which exclude assets in open orders and debt. The calculation on the top bar called "BALANCE" DOES include open orders and debt in the calculation. Using the same word is confusing to many people. People might think that they are holding BTS if the Preferred Unit is BTS and it says "BALANCE: XXX BTS" on the top bar even though it is an approximate calculation of the equivalent VALUE in BTS. Why not change the word to "VALUE"?

2) Calculation: In the "b" tab on the upper left, if you scroll down to Accounts and look on the rightmost column you see it is called "TOTAL VALUE". This is a calculation of balances, orders, and debt and it is appropriately referred to as the account value. This calculation SHOULD be the SAME as the calculation on the upper right but it is not because it lacks a bug as discussed in issue #206. So, why not clone the calculation for "TOTAL VALUE" into the top bar to fix issue #206. They are trying to do the same thing while one is failing and the other is succeeding. I think the calculation is trying to sum the values in the the "Equivalent Value" column in ACCOUNT including open orders but it is failing when all values in Balances are Unknown (because they are too small?). Instead of using this calculation, let's use the one that already works.

This is so simple, change the calculation on the top bar to the same one used in b -> Accounts -> Total Value and rename it to VALUE. This is so easy and makes the program cleaner and clearer.

### PR DESCRIPTION
**Pull request by ariesjia**
fix issue #374  
Updated text  'Balance'  to  'ACCOUNT  VALUE'

### CODE REVIEW

File: web/app/assets/locales/locale-en.json [hunk 1]
```diff
@@ -34,7 +34,8 @@
     "help": "Help",
     "locked_tip": "<br/>Click to login by unlocking your wallet.<br/><br/>Closing the browser window will also log you out.",
     "unlocked_tip": "<br/>Click to log out by locking your wallet.<br/><br/>Closing the browser window will also log you out.",
-    "account_notify": "The active account is now %(account)s"
+    "account_notify": "The active account is now %(account)s",
+    "account_value": "ACCOUNT VALUE"
```
Comment (svk31) [inline web/app/assets/locales/locale-en.json]: Please change ACCOUNT VALUE to normal lowercase "Account value"

File: web/app/assets/locales/locale-en.json [hunk 2]
```diff
@@ -99,7 +100,7 @@
     "show_ignored": "Show hidden accounts",
     "hide_ignored": "Hide hidden accounts",
     "accounts": "Accounts",
-    "estimate_value": "Estimated account value",
+    "estimate_value": "Estimated ACCOUNT VALUE",
```
Comment (svk31) [inline web/app/assets/locales/locale-en.json]: Same as above

File: web/app/assets/locales/locale-en.json (hunk 1 above)
Comment (ariesjia) [reply web/app/assets/locales/locale-en.json]: OK , updated

File: web/app/assets/locales/locale-en.json [hunk 3]
```diff
@@ -99,7 +100,7 @@
     "show_ignored": "Show hidden accounts",
     "hide_ignored": "Hide hidden accounts",
     "accounts": "Accounts",
-    "estimate_value": "Estimated account value",
+    "estimate_value": "Estimated Account value",
```
Comment (svk31) [inline web/app/assets/locales/locale-en.json]: Hate to be a stickler, but please make Account lowercase also.

**Comment by ariesjia** [comment]
Sorry, I did not pay attention to the contributing guide has changed. I merged the branch

**Pull request merged**